### PR TITLE
Add REVIEWING.md

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Add REVIEWING.md
Github-Fixes: #35

Signed-off-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

### CODE REVIEW

File: REVIEWING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- Difficulty is properly set.
+- The topic of the activity / challenge is appropriate.
+- The provided solution is working.
+- Vulnerability and explanations are easy to understand.
+
+## Assets
+
+Check if all of them are used and mentioned in the content.
+If the assets are not original, make sure their license is compatible to [ours](./COPYING.md) and that their source is properly cited.
```
Comment (carrrina) [inline REVIEWING.md:19]:
```suggestion
If the assets are not original, make sure their license is compatible with [ours](./COPYING.md) and that their source is properly cited.
```

File: REVIEWING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Use dash between words (e.g. `image-name.png`).
+- Use only lowercase ASCII characters.
+
+## Content
+
+Make sure that the content is of good quality:
+
+- It follows the general structure of a [session](https://github.com/security-summer-school/template-public/tree/master/session-name-1).
+- Assets are displayed well.
+- The content has at most one sentence per line.
```
Comment (carrrina) [inline REVIEWING.md:31]:
```suggestion
- Each sentence is on a separate line.
```

File: REVIEWING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- The content is easy to understand.
+- Wording and general spelling are correct.
+- Proper formatting is used (such as bold, italic, typewriter formats).
+
+## Working on Pull Requests
+
+You will review contributions as pull requests, following the [PR review guidelines from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+Contributions must follow the [contributing guidelines](CONTRIBUTING.md).
```
Comment (carrrina) [inline REVIEWING.md:39]:
```suggestion
Contributions must follow the [contributing guidelines](./CONTRIBUTING.md).
```

**Review by carrrina — CHANGES_REQUESTED** [review]

File: REVIEWING.md (hunk 3 above)
Comment (gabrielmocanu) [reply REVIEWING.md:39]: By default it will use this path, no need to specific exact path.

Comment (carrrina) [reply REVIEWING.md:39]: Then change the one on line 19 for consistency

Comment (gabrielmocanu) [reply REVIEWING.md:39]: Done.

**Review by carrrina — APPROVED** [review]

**Pull request merged**